exchange/utils.go

```diff
@@ -177,9 +187,9 @@ func ccpaEnabled(account *config.Account, privacyConfig config.Privacy, requestT
 	return privacyConfig.CCPA.Enforce
 }
 
-func extractCCPA(orig *openrtb2.BidRequest, privacyConfig config.Privacy, account *config.Account, aliases map[string]string, requestType config.ChannelType) (privacy.PolicyEnforcer, error) {
+func extractCCPA(orig *openrtb2.BidRequest, privacyConfig config.Privacy, account *config.Account, aliases map[string]string, requestType config.ChannelType, gpp gpplib.GppContainer) (privacy.PolicyEnforcer, error) {
 	// Quick extra wrapper until RequestWrapper makes its way into CleanRequests
-	ccpaPolicy, err := ccpa.ReadFromRequestWrapper(&openrtb_ext.RequestWrapper{BidRequest: orig})
+	ccpaPolicy, err := ccpa.ReadFromRequestWrapper(&openrtb_ext.RequestWrapper{BidRequest: orig}, gpp)
```

Comment:
I think we need to account for the GPPSID. If the SID is missing the CCPA section identifier, I believe we should return the `NilPolicyEnforcer`.

Reply:
Shouldn't the behavior be the same as for GDPR where if the SID is missing the relevant section identifier we fall back to checking the old regs field?
I thought that behavior was implemented for CCPA in `ReadFromRequestWrapper`. If the SID isn't found, the consent string is empty and a `Policy` object is created that may or may not contain a consent string depending on whether it was successfully read from the old regs location and that consent string will determine whether or not we should enforce.

Reply:
This was my thinking as well. Take it from the GPP framework if it exists, and if not, fall back on old behavior. We could revisit in January or later if we should upgrade the old logic for the reality of the GPP framework, but it seems best short term to preserve the old behavior as much as possible as the GPP starts to roll out to the world.

Reply:
Ah my mistake. @SyntaxNode and I just discussed this.
I was looking at this and thinking that was a SID check but it is not:
```
for i, id := range gpp.SectionTypes {
	if id == gppConstants.SectionUSPV1 {
		consent = gpp.Sections[i].GetValue()
	}
}
```